src/components/InputTemplate.jsx

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import React, { useState, useCallback, useEffect } from "react"
+import PropTypes from "prop-types"
+import { Typeahead, withAsync } from "react-bootstrap-typeahead"
+import { getTemplateSearchResults } from "sinopiaSearch"
+
+const AsyncTypeahead = withAsync(Typeahead)
```

Comment:
Refactored an input for templates into its own component since used in 2 locations.